tc/tc_util.c

```diff
@@ -191,6 +191,61 @@ static const struct rate_suffix {
 	{ NULL }
 };
 
+int parse_percent_rate(char *rate, const char *str, char *dev)
+{
+	long max_rate_bits;
+	int ret;
+	double perc, rate_bits;
+
+	if (dev == NULL) {
+		fprintf(stderr, "No device specified; please specify device\n"
+);
```

Comment:
Let's make this `"No device specified; specify device to rate limit by percentage\n"`.

Also, move the `);` onto the previous line.